src/app/lib/config/services/persian.js

```diff
@@ -66,15 +66,16 @@ export const service = {
     showRelatedTopics: true,
     podcastPromo: {
       title: 'پادکست',
-      brandTitle: 'داستان انقلاب به روایت بی‌ بی سی',
-      brandDescription: 'مجموعه داستان انقلاب از آرشیو رادیوی بی بی سی',
+      brandTitle: 'به عبارت دیگر',
+      brandDescription:
+        'گفتگو با چهارهای سیاسی، هنری و ادبی از تلویزیون فارسی بی بی سی.',
       image: {
-        src: 'https://ichef.bbci.co.uk/images/ic/$recipe/p0703j1b.jpg',
-        alt: 'داستان انقلاب به روایت بی‌ بی سی',
+        src: 'http://ichef.bbci.co.uk/images/ic/$recipe/p036kbm2.jpg',
```

Comment:
should this be `https`?

Reply:
```suggestion
        src: 'https://ichef.bbci.co.uk/images/ic/$recipe/p036kbm2.jpg',
```

Reply:
Ye.  good spot - when we receive links from Editorial they come in without. We have had to update recent configs with that recently. Will get it updated. Thanks.

Reply:
done!